proto/log.proto

```diff
@@ -48,7 +48,6 @@ message Log {
   int32 rootLogId = 8;
   // Array of minified Run objects.
   repeated MinimalRun runs = 9;
-  repeated Subsystem subsystems = 10;
   LogSubtype subtype = 11;
```

Comment:
I cannot add a comment on the line 68 so I am adding it here.
Should you not also remove `LOG_SUBTYPE_SUBSYSTEM` from L68?

Reply:
Done